recipes/qr-code-generator/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.0)
+project(test_package)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
```suggestion
```

Reply:
Removing this will cause the test build fail, see [this build](https://github.com/conan-io/conan-center-index/pull/1817#issuecomment-638132755)